### DISCUSSION TITLE
Help configuring devtools

### DISCUSSION THREAD

**Discussion by b-mounir-dev** [nuxt/devtools#656] [Q&A]
When opening devtools, it says "Not connected to the client app, showing server-side data only. Use the embedded mode for full features."
What should I do ? I couldn't find helpful information about this.

*Screenshot*
<img width="546" alt="image" src="https://github.com/nuxt/devtools/assets/136018527/de054952-76e0-4f69-96cd-67414d7a8f7d">

**Comment by MortenROSE** [comment]
I'm facing the same with our nuxt app that is running inside a docker container and gets proxy_pass'ed when visiting specific routes in our php webapp, e.g. if you visit `example.com/p/product-123` the nuxt-app gets called.

How do you setup the embedded mode?
